src/_data/events.yaml

```
@@ -1,5 +1,10 @@
 - type: "Webinar"
-  title: "Building a foundation for Enterprise Agility and Process Optimization"
+  title: "Building a Foundation for Enterprise Agility and Process Optimization"
   buttonText: "Learn more"
   link: "/webinars/2024/building-a-foundation-for-enterprise-agility-and-process-optimization/"
-  expire: "2024-06-26T16:00:00Z"
\ No newline at end of file
+  expire: "2024-06-26T16:00:00Z"
+  - type: "Whitepaper"
```

```suggestion
- type: "Whitepaper"
```
Indentation is critical on these files, it had a couple of extra spaces

---

This is the reason the tests are failing